pkg/validators/am0007_test.go

```diff
@@ -14,19 +15,25 @@ func init() {
 type TestAM0007 struct{}
 
 func (v TestAM0007) Name() string {
-	return AM0007.Name
+	return validators.AM0007.Name
 }
 
 func (v TestAM0007) Run(mb types.MetaBundle) types.ValidatorResult {
-	return AM0007.Runner(mb)
+	return validators.AM0007.Runner(mb)
 }
 
-var (
-	testBundle = testutils.NewBundle("random-bundle", "../../internal/testdata/assets/am0007/csv.yaml")
-)
```

Comment:
Any reason behind shifting from calling this function once to calling it twice individually under SucceedingCandidates() and FailingCandidates()? :)

Reply:
Same issue regarding error handling here. Declaring this globally within the package makes error handling difficult and leads to using unnecessary panics. Calling this within function scope allows for errors to be passed gracefully back to the calling package.